web-api/src/app.ts

```diff
@@ -1008,7 +1001,6 @@ app.get(
   );
   app.get('/users-by-role', lambdaWrapper(getAllUsersByRoleLambda));
   app.get('/users', lambdaWrapper(getUserLambda));
-  app.post('/users', lambdaWrapper(createUserLambda));
```

Comment:
We use this API to create Court users today. Why did case-sensitivity necessitate the removal of this endpoint?

Reply:
Good question. 
Short answer: It was not required, but felt like a good idea. Creating users was changed from hitting an endpoint to running a script.

Longer answer: We saw that our pipeline was flakey when creating judges and creating court users. It would just randomly fail, and sometimes it would fail and then log the password in the error. When re-run with no changes it would pass. When we investigated what was happening we thought it was strange that we had a public endpoint that basically no one was allowed to hit but the dev team. If that was the case why have it be an endpoint? Why not make it a script?

So we decided to delete that endpoint, interactor, and code because it felt strange to have a public route for creating a user that was not meant to be hit by the public, we had multiple paths for creating a user which needed to retain context of lowercasing, remove methods off of appContext that were not meant to be used by the rest of the application, reduced pipeline complexity as we do not need to activate a user to to then have that user create more users we can just run a script, and (subjectively) make the code simpler for what was happening when creating judgeUsers and testUsers. 

Totally open to input though.